src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.style-1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.style-2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
```

Comment:
The name of the element class should make a sense. It should describe what the role of the element is on this page or describe its state.